### ISSUE TITLE
Exclude devDependencies from the pack

### ISSUE THREAD

**Issue by jirutka** [ripeworks/workspace-pack#4]
workspace-pack currently includes even `devDependencies` in the generated zip file, IMO only `dependencies` should be included.

**Comment by tamagokun** [comment]
I agree, I think a test is in order for this. I'm setting up `npm-remote-ls` to not look at dev deps and optional deps: https://github.com/ripeworks/workspace-pack/blob/master/bin/index.js#L9

Thanks for reporting!

**Comment by tamagokun** [comment]
I've added a test for this, and it _seems_ to be working. Can you give me an example of it failing? I could add it to the test case.

https://github.com/ripeworks/workspace-pack/blob/master/test/index.js#L4-L13
